x-pack/plugins/index_management/public/application/index.tsx

```diff
@@ -64,24 +65,38 @@ export const renderApp = (
     executionContext,
   };
 
-  render(
+  return (
```

Comment:
nit: we're encouraging people to use [`KibanaRenderProvider`](https://github.com/elastic/kibana/blob/main/packages/react/kibana_context/render/README.mdx) with these.